src/collectives/device/common.h

```diff
@@ -9,7 +9,7 @@
 
 #include "collectives.h"
 #include "devcomm.h"
-
+#include <stdio.h>
```

Comment:
Why use stdio?  This seems like it would be for printf?  Maybe remove and use std::cout?

Reply:
Yup, will do.